src/lib.rs

```diff
@@ -66,15 +69,31 @@ use {
   prometheus_metrics::serve_prometheus,
 };
 
+#[derive(Parser, Debug)]
+struct CmdArgs {
+  #[arg(long, default_value_t = false)]
+  /// if you start multiple lemmy server instances set this to true on all but one of them
+  disable_scheduled_tasks: bool,
+  /// set to false to disable the http server
+  #[arg(long, default_value_t = true, action=ArgAction::Set)]
+  http_server: bool,
+  /// set to false to disable the outgoing federation in this process
+  #[arg(long, default_value_t = true, action=ArgAction::Set)]
+  send_activities: bool,
+  /// the index of this outgoing federation process (1-based). only useful if you want to split federation work into multiple servers
+  #[arg(long, default_value_t = 1)]
+  activity_worker_index: i32,
+  /// how many outgoing federation processes you are starting in total
+  #[arg(long, default_value_t = 1)]
+  activity_worker_count: i32,
```

Comment:
Worker count sounds like threads in the old implementation. Better to call it process count if thats what it is.

Reply:
Yeah, makes sense. Updated to `federate_process_index/count`